README.md

```diff
@@ -127,6 +127,40 @@ It integrates naturally with Python and Ruby test frameworks.
 It is fast! It will not increase the time of your tests beyond the acceptable.
 
 
+## How to build
+
+These instruction are currently tested on Ubuntu 14.04. Other distros might work just fine, but they
+were never tested, so 14.04 should give you the most trustworthy results.
+
+Getting the dependecies:
+
+   ```sh
+   pip install pyroute2
+   sudo apt-get install swig python-dev
+   ```
+
+Building:
+
+   ```sh
+   git clone https://github.com/datacom-teracom/l2tester.git
+   cd l2tester
+   make python # e.g build the Python wrappers
+   sudo make install # install Python module
+   ```
+
+To send packets through the computer interface without needing sudo permissions for every user, it
+is recommended to set permissions for the Python application to access the network interfaces:
+
+   ```sh
+   sudo setcap cap_mac_admin,cap_net_raw,cap_net_admin=eip /usr/bin/python2.7
+   ```
+
+Replace `python2.7` by your local version of Python.
+
```

Comment:
And here, I would prefer if we create another chapter "Building with other wrappers".

And also mention that other dependencies may be required.